CHANGELOG.rst

```diff
@@ -5,6 +5,10 @@ Changelog
 unreleased
 ==========
 
+0.0.32 (2022-01-05)
```

Comment:
Your PR is an individual change not a release PR, the changelog entry should be in unreleased